### PR TITLE
fix: correct url in openapi specs

### PR DESCRIPTION
**Pull request by vatanasov**
refs: https://github.com/aeternity/ae_mdw/issues/1759
Decided to go with the second proposed solution, it would seem to work for the different cases and it would not require additional configurations.

### CODE REVIEW

**Comment by sborrazas** [comment]
Have you tried this with the staging/prod setup? I don't think this works as intended, we are running the middleware app under the `/mdw/` namespace via an nginx proxy, which removes the `/mdw/` from the path to call the middleware app, which is how the middleware responds

**Comment by vatanasov** [comment]
> Have you tried this with the staging/prod setup? I don't think this works as intended, we are running the middleware app under the `/mdw/` namespace via an nginx proxy, which removes the `/mdw/` from the path to call the middleware app, which is how the middleware responds

Ah, it seems you are correct, I will try something else then, thank you.

File: Dockerfile [hunk 1]
```diff
@@ -98,6 +98,7 @@ COPY config/runtime.exs config/
 
 # Generate swagger V2 file
 RUN mix run --no-start -e 'IO.puts(Mix.Project.config[:version])' >AEMDW_VERSION
+ARG PATH_PREFIX
```
Comment (sborrazas) [inline Dockerfile:101]: I don't think ARGs in dockerfiles are env variables. This wouldn't be accessed on the swagger script

**Review by sborrazas — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (vatanasov) [reply Dockerfile:101]: They are, for `RUN` commands: https://docs.docker.com/reference/dockerfile/#using-arg-variables . If you wanna test it, you can rebuild the docker container using the `docker-compose-dev.yml` included in the PR

Comment (sborrazas) [reply Dockerfile:101]: You're right, it works the same! sorry, merging

**Review by sborrazas — APPROVED** [review]

**Pull request merged**